RubberduckTests/Inspections/VariableNotUsedInspectionTests.cs

```diff
@@ -157,6 +157,19 @@ public void VariableNotUsed_AssignedMultpleTimesButNeverReferenced_ReturnsResult
             Assert.AreEqual(1, InspectionResultsForStandardModule(inputCode).Count());
         }
 
+        [Test]
+        [Category("Inspections")]
+        public void UndeclaredVariableNotUsed_NoResults()
```

Comment:
Do we already have a test on the `VariableNotUsedInspection` to raise for an undeclared variable? I don't see such test in my quick check and we should cover this.

Reply:
@bclothier I don't understand, undeclared variables should _not_ be issuing `VariableNotUsedInspection` results anymore, this test covers that.

Reply:
Sorry now I see that I got confused and thought this was testing the `UndeclaredVariableInspection`.